build.yaml

```diff
@@ -1184,6 +1184,13 @@ libs:
   - grpc_test_util
   - grpc++_test_util
   - grpc++
+- name: thrift_util
+  build: all
+  language: c++
+  public_headers:
+  - include/grpc++/impl/codegen/thrift_serializer.h
+  - include/grpc++/impl/codegen/thrift_serializer_inl.h
+  - include/grpc++/impl/codegen/thrift_utils.h
```

Comment:
As you've already seen, this doesn't make sense as a headers-only library. I understand that your purpose is to get it included as part of the installation. You should follow the same steps used for proto_utils.h ; define the name, don't put a build: all on it, and make sure that it gets added to some other dependence chain.